messages.rb

```diff
@@ -276,6 +289,92 @@ def send_list(event)
   event.attach_file(File::open(tmpfile))
 end
 
+def send_community(event)
+  msg = event.content
+  tabs = parse_tabs(msg)
+  tabs.empty? ? ftabs = "" : ftabs = format_tabs(tabs)
+  tabs = (tabs.empty? ? [:SI, :S, :SL, :SU, :SS, :SS2] : tabs)
+
+  levels = Score.where(rank: 0, highscoreable_type: "Level").includes(:level).where(levels: {tab: tabs})
+  episodes = Score.where(rank: 0, highscoreable_type: "Episode").includes(:episode).where(episodes: {tab: tabs})
+  lvls = levels.count
+  eps = episodes.count
+  tls = levels.map{|s| s.score}.sum
+  tes = episodes.map{|s| s.score}.sum
+  if !(tabs&[:SS, :SS2]).empty? # in order to compare lvls and eps we have to exclude secrets
+    tlsno = Score.where(rank: 0, highscoreable_type: "Level").includes(:level).where(levels: {tab: tabs}).where.not(levels: {tab: [:SS, :SS2]}).map{|s| s.score}.sum
+    ttlsno = "Total level score (w/o secrets): #{tlsno}\n"
+  else
+    tlsno = tls
+    ttlsno = ""
+  end
+  diff = tlsno-4*90*eps-tes
+  ttls = "Total level score: #{tls}\n"
+  ttes = "Total episode score: #{tes}\n"
+  tdiff = "Difference: #{"%.3f" % [diff]}\n"
+  ttlsavg = "Average level score: #{"%.3f" % [tls/lvls]}\n"
+  ttesavg = "Average episode score: #{"%.3f" % [tes/eps]}\n"
+  avgdiff = "Average difference: #{"%.3f" % [diff/eps]}\n"
+
+  event << "Community's total #{ftabs}scores #{Time.now.strftime("on %A %B %-d at %H:%M:%S (%z)")}:\n```#{ttls}#{ttlsno}#{ttes}#{tdiff}\n#{ttlsavg}#{ttesavg}#{avgdiff}```"
```

Comment:
This `strftime` with that format should probably be broken out into its own `format_time` or something like that since it's repeated a few times now.